deploy/compose/docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
     container_name: admin
     hostname: admin
     ports:
-      - "${HOPR_ADMIN_PORT}:80"
+      - "${HOPR_ADMIN_PORT}:4677"
```

Comment:
This is not necessary, is it? I though that the default was 80 on the service.

Reply:
It's necessary since this PR https://github.com/hoprnet/hopr-admin/pull/630